src/jvm/main/org/jetbrains/kotlinx/lincheck/IdeaPlugin.kt

```diff
@@ -229,9 +229,15 @@ private fun constructTraceForPlugin(failure: LincheckFailure, trace: Trace): Arr
                     is ObstructionFreedomViolationExecutionAbortTracePoint -> 6
                     else -> 0
                 }
+                
+                val readOrWriteType = when (event) {
+                    is ReadTracePoint -> event.valueType
+                    is WriteTracePoint -> event.valueType
+                    else -> ""
+                }
 
                 if (representation.isNotEmpty()) {
-                    representations.add("$type;${node.iThread};${node.callDepth};${node.shouldBeExpanded(false)};${eventId};${representation};${location};${locationId}")
+                    representations.add("$type;${node.iThread};${node.callDepth};${node.shouldBeExpanded(false)};${eventId};${representation};${location};${locationId};[${readOrWriteType}];false")
```

Comment:
Please update the documentation of the `constructTraceForPlugin` with the description of the new parameters of trace points passed to the plugin.

Reply:
done!